nano/node/blockprocessor.hpp

```diff
@@ -53,6 +53,8 @@ class block_processor final
 	nano::vote_generator generator;
 	// Delay required for average network propagartion before requesting confirmation
 	static std::chrono::milliseconds constexpr confirmation_request_delay{ 1500 };
+	// Threshold for not requesting confirmation when there are too many active elections
+	static size_t constexpr confirmation_request_cutoff{ 1000 };
```

Comment:
Is this duplication of `max_broadcast_queue` or is the same value coincidental ? Asking since  max_broadcast_queue is used in computation of the extra delay, removed by this PR.

Reply:
Coincidence, but good catch! We can still tweak `max_broadcast_queue` if we see fit